data/remnant missions.txt

```diff
@@ -1252,3 +1252,240 @@ mission "Remnant: Expanded Horizons Quarg 3"
+	on complete
+		conversation
+			`As promised, there is a team of researchers waiting when you settle onto the designated landing pad. One comes over to hand you <payment> while the rest carefully unload the broken drive and start hauling it back to their lab.`
+			`	"Our thanks, Captain <last>. Having a partially functional drive like this will be very beneficial to our efforts." His voice cracks a bit, as if unused to being used much.`
+			choice
+				`	"Great! I will find more."`
+					goto end
+				`	"What are you studying?"`
+			`	"Our previous research into Jump Drives determined that these drives are unsurprisingly multi-part systems. We know the outer part that we can see and handle is mostly the machinery for generating some kind of containment field or pocket. As to how it works or why it is needed..." his chant trails off and he makes a gesture you suspect may be the equivalent of a shrug. "Anyways, our team is focused on the nature of this outer system. It is our hope that - damaged as it is - its continued functionality might allow us to look a little farther inside its inner workings to figure out exactly what that field does and how it is used."`
+			label end
+			`	"Well, my compatriots finished unloading rather quickly, so I should be off. For all our sakes, I hope you find more of these for us."`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 3"
+	name "Broken Jump Drive Delivery"
+	description "The researchers on <planet> need a broken jump drive for testing."
+	to offer
+		has "Remnant: Broken Jump Drive 2: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+		attributes outfitter
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As you arrive in the spaceport concourse the director looks up and waves you over.`
+			`	"Good timing, Captain <first>," he sings, "we have a team of researchers on <planet> who have requested a broken jump drive. Could you deliver one to them?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent. The recent discovery of these broken drives has spurred discussion and excitement in various quarters. We have had teams researching jump drives for about at least a century now without a lot of success. I'm told they are rather intricate, but your finds are opening new avenues of investigation. That being said, I'll notify the team on <planet> to expect you."`
+				accept
+			label refuse
+			`	He looks disappointed, and taps something on his screen. "OK, I'll file that request away for now. Come back if you change your mind."`
+				defer
+	on complete
+		conversation
+			`The team waiting for you on <planet> is definitely eager to begin. You barely have the hatch open before they are chivying their camel-pulled wagon up the ramp. They load the broken jump drive surprisingly quickly despite their delicate movements.`
```

Comment:
Is "chivying" the right word? I'd never heard this before and wiktionary seems to suggest that it means to hunt, chase, or harass. might suggest replacing this with `...before they drive their camel-pulled wagon...`

Reply:
I'm not sure about wiktionary, but the google dictionary gives it as `tell (someone) repeatedly to do something. "an association that chivvies government into action" `

Mirriam Websters: ` 2 : to move or obtain by small maneuvers "chivy an olive out of a bottle"` 

It isn't exactly a common word, but I attribute the fact that one of the main uses for it around these parts is in regards to herding cattle, and herding cattle isn't exactly a common activity anymore.

Reply:
Wow, you learn something every day.

Reply:
Do all planets have camels? As this mission can complete anywhere, but only one planet description mentions camels.

*Imagines camels unloading a jump drive in the middle of a blizzard.*

Reply:
Yep, they all do. 

Fun fact: Bactrian camels naturally migrate over a span of territory that can vary from -40C to +40C. So yeah, I'd say they are probably just as effective at unloading stuff in a blizzard as any other time. ([source:bactrian wikipedia article](https://en.wikipedia.org/wiki/Bactrian_camel#Natural_habitat)